app/client/ui/MakeCopyMenu.ts

```diff
@@ -362,3 +502,31 @@ const cssSpinner = styled('div', `
 const cssCheckbox = styled(labeledSquareCheckbox, `
   margin-top: 8px;
 `);
+
+const cssAttachmentsDownloadRow = styled('div', `
+  margin: 16px 0;
+  display: flex;
+  gap: 16px;
+  align-items: center;
+`);
+
+const cssArchiveFormatSelect = styled(select, ``);
+
+const cssDownloadAttachmentsButton = styled(bigPrimaryButtonLink, `
+  text-wrap: nowrap;
+`);
+
+// Prevents the div from expanding the parent and makes it only use available space instead.
+const cssEagerWrap = styled('div', `
+  min-width: 100%;
```

Comment:
Very hacky, didn't know this trick. Wdyt about modern alternative ?
`contain: inline-size;`

Reply:
Oooh, this is nice. Seems to do the same job in practice. Much cleaner than the hack above too.

Weird that this didn't come up when I was searching for options here!